result.go

```diff
@@ -39,15 +58,25 @@ func (r VerifyResult) ShouldAccept() bool {
 		if r.strict { // If Strict mode is enabled, we do not accept any captcha if there was an error.
 			return false
 		}
-		if errors.Is(r.err, ErrVerificationRequest) || errors.Is(r.err, ErrVerificationFailedDueToClientError) { // Failure to talk to Friendly Captcha verification API or client error (e.g. wrong API key)
+		if errors.Is(r.err, ErrVerificationRequest) ||
```

Comment:
Nit: I'd probably try break on the `||` here personally if the issue is long lines, I think the second `errors.Is` is hard to read now in that format. If the current version is added by golines then maybe it makes sense to keep as-is.

Reply:
Yeah this is what `golines` made out of it.. Maybe there is some way to configure it, but for now I'll just follow its defaults